docker-compose.yml

```
@@ -1,24 +1,36 @@
 services:
-
   ensnode:
-      build:
-        dockerfile: ./apps/ensnode/Dockerfile
-      image: namehash/ensnode
-      ports:
-        - "42069:42069"
-      env_file:
-      - ./apps/ensnode/.env.local
-      environment:
-        # Override DATABASE_URL from .env.local to refer to docker compose postgres instance
-        DATABASE_URL: postgresql://postgres:password@postgres:5432/ponder
-        ENSRAINBOW_URL: https://ensrainbow:3000
-      depends_on:
-        - ensrainbow
-        - postgres
+    container_name: ensnode
+    image: namehash/ensnode
+    build:
+      dockerfile: ./apps/ensnode/Dockerfile
+    ports:
+      - "42069:42069"
+    env_file:
+    - ./apps/ensnode/.env.local
+    environment:
+      # Override DATABASE_URL from .env.local to refer to docker compose postgres instance
+      DATABASE_URL: postgresql://postgres:password@postgres:5432/ponder
+      # Override ENSRAINBOW_URL to point to docker compose ensrainbow
+      ENSRAINBOW_URL: http://ensrainbow:3223
```

We might need to set the ponder schema so the production build could go ahead:
```suggestion
      DATABASE_SCHEMA: local
      ENSRAINBOW_URL: http://ensrainbow:3223
```